assets/app/views/_triggers.html

```diff
@@ -1,11 +1,25 @@
 <div class="builds" ng-repeat="trigger in triggers">
-  <div ng-repeat="build in trigger.builds" ng-if="(build.status != 'Complete' && build.status != 'Failed') || (build.metadata.creationTimestamp | ageLessThan : 5 : 'minutes')" class="build animate-repeat osc-object" kind="Build" resource="build">
+  <div ng-repeat="build in trigger.builds" ng-if="build.status == 'New' || build.status == 'Pending' || build.status == 'Running' || (build.metadata.creationTimestamp | ageLessThan : 5 : 'minutes')" class="build animate-repeat osc-object" kind="Build" resource="build">
```

Comment:
Probably worth separating the check if the build is running into a filter ("isBuildInProgress", etc) that checks new, pending, running